src/tools/mergeRefs.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @param {...Ref<Element>} refs List of React refs to merge.
+ * @returns {Ref<Element>} Merged React ref.
+ */
+const mergeRefs = (...refs) => el => {
+  refs.forEach(ref => {
+    // https://github.com/facebook/react/issues/13029#issuecomment-410002316
```

Comment:
thanks for providing issue reference here. wonder if there is any more discussion on why `RefObject` type definition is read only

Reply:
Yeah we need to monitor how the discussion goes. I wish React had better consideration of this use case.